### PR TITLE
Babyhint param count updates

### PR DESCRIPTION
**Pull request by EthanLuisMcDonough**
### High-level description of the changes
This pull request ~addresses issue 640 and~ fixes the param counts present in [babyhint.js](https://github.com/Khan/live-editor/blob/master/js/output/pjs/babyhint.js)

### Are there performance implications for this change?
As far as I'm aware, no.

### Have you added tests to cover this new/updated code?
Yes.

### Risks involved
I don't see any risks present in these changes.

### Are there any dependencies or blockers for merging this?
No.

### How can we verify that this change works?
I've added tests.

### CODE REVIEW

**Comment by pamelafox** [comment]
Hello! If you are intending for this PR to be reviewed, please write a description. I've just created a template you can fill out here:
https://raw.githubusercontent.com/Khan/live-editor/master/PULL_REQUEST_TEMPLATE.md

**Comment by EthanLuisMcDonough** [comment]
Thanks for the template.  I've added some tests as well as a description.

**Comment by pamelafox** [comment]
Thanks for the additional description!
Can you separate this into 2 PRs, one for each distinct fix? (The AST versus the babyhint)
It's nice to have each PR be a single-feature PR when possible.

**Comment by EthanLuisMcDonough** [comment]
Thanks for the feedback.  I removed the AST fix for now, but I'll make another PR once this one gets merged.

File: build/js/live-editor.output_pjs.js [hunk 1]
```diff
@@ -1731,10 +1731,10 @@ var BabyHint = {
         "curve": 8,
         "cursor": [0, 1, 2, 3],
         "endShape": [0, 1],
-        "dist": 4,
-        "fill": [1, 3, 4],
+        "dist": [4, 6],
+        "fill": [1, 2, 3, 4],
         "filter": [1, 2],
-        "get": [2, 3, 4, 5],
+        "get": [0, 2, 3, 4, 5],
```
Comment (pamelafox) [inline build/js/live-editor.output_pjs.js]: I don't use get() much myself but I know it's popular in the community. Is the performance ok when you call get() on an entire image?

Comment (EthanLuisMcDonough) [reply build/js/live-editor.output_pjs.js]: Smaller `get` snapshots are faster, but [some community members want to be able to invoke `get` without passing in any arguments](https://khanacademy.zendesk.com/hc/en-us/community/posts/115005702387-Documentation-for-get-x-y-w-h-).

**Comment by EthanLuisMcDonough** [comment]
Ok, I signed it.

**Comment by pamelafox** [comment]
Okay, thanks, looks good! I'm merging this now. I'll probably deploy next in 1-3 weeks.

**Pull request merged**